google/cloud/aiplatform/datasets.py

```diff
@@ -118,11 +120,13 @@ def create(
                 is defined as an OpenAPI 3.0.2 Schema Object. The schema files
                 that can be used here are found in gs://google-cloud-
                 aiplatform/schema/dataset/metadata/.
-            source: Optional[Sequence[str]]=None:
+            gcs_source: Optional[Sequence[str]]=None:
                 Google Cloud Storage URI(-s) to the
                 input file(s). May contain wildcards. For more
                 information on wildcards, see
                 https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames.
+            bq_source: Optional[str]=None:
+                BigQuery URI to the input dataset.
```

Comment:
What's meant by URI? Could you provide an example?

Also, does "dataset" refer to a BigQuery dataset or table?

Reply:
BigQuery Table, let me make it more explicit.

It's one of these, what's our term for this?

`bq://bigquery-public-data.austin_311`

Reply:
You can see an example here: https://cloud.google.com/automl/docs/reference/rpc/google.cloud.automl.v1beta1#google.cloud.automl.v1beta1.BigQuerySource

Reply:
Spoke with the team and they said the format is quite common. Have you seen another version?

Reply:
BQ itself doesn't use this prefix format in any place I'm aware.  Storage resources have multiple formats (TableReference), OP style string names (projects/p/datasets/d/tables/t), sql identifiers (p.d.t) depending on where they're getting consumed.

I can't find any reference to it in BQ code, but there's lots of references from aiplatform things, and related things in the ML space (kaggle).